### ISSUE TITLE
Optimise PSI tests by splitting the tests

### ISSUE THREAD

**Issue by mohammedzee1000** [openshift/odo#4328]
This is dependent on https://github.com/openshift/odo/issues/4056 
- [x] Split run script into setup + run. This will require minor updates to ci-firewall codebase
- [x] Create new send queues corresponding th jenkins jobs in rabbit mq
- [x] Create more jenkins jobs for each split we want to do for eg 1 for unit and 1 for int and e2e
- [ ] Split run_all_tests accordingly

**Comment by anandrkskd** [comment]
Once ths PR #4063 is merged, We can start working on this issue.

1st objective of AC is half completed as as minor update to CI-firewall codebase is already done, the part which is left is dividing the [run script](https://github.com/openshift/odo/blob/3524e62d1d66cf95313d655c46d19134c5c4db4a/scripts/run_all_tests.sh) into two scripts `setup` and `run` script. 

2nd objective of AC requires creating queues on rabbitmq  which is hosted on public cloud 

3rd Jenkins job need to be created using  jenkins dashboard and configure it such that it splits unit test and integration and e2e tests.

4th as we will devide the jobs runscript needs to be divided into multiple part possibly two 1 for unit test 2nd for e2e and integration

**Comment by dharmit** [comment]
> 1st objective of AC is half completed as as minor update to CI-firewall codebase is already done,

Can you point to an issue or PR that addressed this?


> 2nd objective of AC requires creating queues on rabbitmq which is hosted on public cloud

When does this need to be created? After moving to PSI or before?

**Comment by anandrkskd** [comment]
> When does this need to be created? After moving to PSI or before?
working on this issue will start once the POC [PR](https://github.com/openshift/odo/pull/4063) is merged.

**Comment by prietyc123** [comment]
I think we have already separated out the CI scripts accordingly and following the same approach for upcoming job. IMO we can close this issue safely. @mohammedzee1000 WDYT?
